### PR TITLE
added logging to TransactionProxy.rollback in case of exception

### PR DESCRIPTION
**Pull request by gurbuzali**

### CODE REVIEW

File: hazelcast-client/src/main/java/com/hazelcast/client/proxy/txn/TransactionProxy.java [hunk 1]
```diff
@@ -121,8 +123,8 @@ void rollback() {
             try {
                 ClientMessage request = TransactionRollbackCodec.encodeRequest(txnId, threadId);
                 invoke(request);
-            } catch (Exception ignored) {
-                EmptyStatement.ignore(ignored);
+            } catch (Exception exception) {
+                logger.warning("Exception while rolling back the transaction!!!", exception);
```
Comment (jerrinot) [inline hazelcast-client/src/main/java/com/hazelcast/client/proxy/txn/TransactionProxy.java]: do we really need 3x `!!!` ? It looks a bit hysterical, for my taste anyway :)

Comment (gurbuzali) [inline hazelcast-client/src/main/java/com/hazelcast/client/proxy/txn/TransactionProxy.java]: that's a habit of mine while adding logs :) let me remove

Comment (jerrinot) [inline hazelcast-client/src/main/java/com/hazelcast/client/proxy/txn/TransactionProxy.java]: there is also the same issue in `XATransactionProxy`

Comment (gurbuzali) [inline hazelcast-client/src/main/java/com/hazelcast/client/proxy/txn/TransactionProxy.java]: done

**Comment by jerrinot** [comment]
:+1:

**Comment by jerrinot** [comment]
looks good. it should be ported to the `master` branch as well.

**Pull request merged**